### PR TITLE
Update setup docs to mention cloud-auth command

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

Blocked by
- https://github.com/mattermost/mattermost-plugin-jira/pull/928

### CODE REVIEW

File: docs/setup/configuration.md [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
+2. Click **Configure** next to the plugin's name.
+3. At the top of the page set **Enable Plugin** to **True**.
+4. Choose **Save** to enable the Jira plugin.
+5. In the Mattermost chat UI, run `/jira setup` to start configuring the plugin.
 
 ### Step 2: Install the plugin as an application in Jira
 
-To allow users to [create and manage Jira issues across Mattermost channels](../end-user-guide/using-jira-commands.md), install the plugin as an application in your Jira instance. For Jira Server or Data Center instances, post `/jira instance install server <your-jira-url>` to a Mattermost channel as a Mattermost System Admin, and follow the steps posted to the channel. For Jira Cloud, post `/jira instance install cloud <your-jira-url>`.
+To allow users to [create and manage Jira issues across Mattermost channels](../end-user-guide/using-jira-commands.md), install the plugin as an application in your Jira instance. For Jira Server or Data Center instances, post `/jira instance install server <your-jira-url>` to a Mattermost channel as a Mattermost System Admin, and follow the steps posted to the channel. For Jira Cloud, post `/jira instance install cloud-oauth <your-jira-url>`.
```
Comment (mickmister) [inline docs/setup/configuration.md:13]: Changes other than this one are just cleaning up the file

**Comment by Kshitij-Katiyar** [comment]
@mickmister I have done some minor changes according to our QA's comments. To merge this and include in our latest release we need to fix the CI here and one possible solution for this is to sync the branch with its base branch. But i am not able to understand these doc branches like ```[jira-docs-3.0.1](https://github.com/mattermost/mattermost-plugin-jira/tree/jira-docs-3.0.1)``` and similar branches. Can you please let me know how this PR is created and possibly fix the CI.

**Comment by mickmister** [comment]
@Kshitij-Katiyar Since the recent PR https://github.com/mattermost/mattermost-plugin-jira/pull/948, we have deprecated the Gitbooks docs, and are instead favoring the README to hold all of the information. So we just need to make sure the README is updated on master.

cc @cwarnermm Do you think we should remove the `docs` folder to avoid confusion on where documentation lives? Or does it play a role in some capacity?

**Comment by mickmister** [comment]
Closing this since we are now putting docs in the README instead of Gitbooks

**Comment by cwarnermm** [comment]
@mickmister - I'm in the process of removing the ``docs`` folders in the GitHub repos for community-supported plugins. That ``docs`` folder no longer plays a role since it was created as part of GitBook synchronization functionality, and we're no longer hosting the source of truth via GitBook. (Note: We are replacing that GitBook content with redirects to help visitors find the README and plugin docs.)

**Pull request closed without merging**